automatic/nodejs.install/nodejs.install.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>nodejs.install</id>
+    <title>Node JS (Install)</title>
+    <version>7.4.0</version>
+    <authors>Node.js Foundation</authors>
+    <owners>chocolatey, Rob Reynolds</owners>
+    <summary>Node JS - Evented I/O for v8 JavaScript.</summary>
+    <description>
+Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.    
+
+This package runs the official Node JS installer, resulting in Node.exe and NPM being installed under Program Files, and added to the SYSTEM PATH environment variable.
+    </description>
```

Comment:
@AdmiringWorm I couldn't find a feature list on their website. Should I write something myself in this case or omitt it?